doc/nrf/device_guides/nrf52/testing.rst

```diff
@@ -0,0 +1,36 @@
+.. _nrf52_computer_testing:
+
+Testing with multiple nRF52 Series devices
+##########################################
```

Comment:
Do we really need this page? It is repeating a scenario from a sample Testing section, and somewhat repeating https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/gsg_guides/nrf52_gs.html#testing_the_sample .

Reply:
I don't think it makes sense to keep this page here. There are other samples that have testing scenarios for 2+ development kits, yet here we specifically refer to Peripheral UART. This whole page should rather be a subsection of https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/samples/bluetooth/peripheral_uart/README.html#testing

This whole page was once moved from Infocenter, where it maybe had its purpose, but in the NCS it is redundant.
Re https://github.com/nrfconnect/sdk-nrf/commit/8796a90ba355899c6eafbaf3c68f2488049277b8#diff-5c5fa22b8c1b99fc19b85717fe5a405395e647a4a0f82a7326744c56e35cd4b2